bridgenode/flatfileworker.go

```diff
@@ -175,6 +202,55 @@ func (ff *flatFileState) ffInit() error {
 	return nil
 }
 
+func (ff *flatFileState) writeUndoBlock(ud accumulator.UndoBlock) error {
```

Comment:
Could we change the undoblock variable name here to `ub`? Since `ud` is used in other parts of the codebase to mean Utreexo Data

Reply:
Okay Ill make the changes